homeassistant/components/notify/yessssms.py

```diff
@@ -38,14 +39,35 @@ def __init__(self, username, password, recipient):
         from YesssSMS import YesssSMS
         self.yesss = YesssSMS(username, password)
         self._recipient = recipient
+        # pylint: disable=protected-access
+        _LOGGER.debug(
+            "initialized; library version: %s", self.yesss._version)
```

Comment:
This should be publicly exposed from the upstream lib, or this debug line can just be removed.